src/FiltrationGroupComponent/components/HorizontalBracket/HorizontalBracket.tsx

```diff
@@ -36,13 +36,17 @@ export const HorizontalBracket: React.FC<HorizontalBracketProps> = ({
 								)
 							) {
 								positionTop = height + minHeight / 2;
-							}
-							if (
+							} else if (
 								item.element.classList.contains(
 									SearchClasses.KitFiltrationGroup
 								)
 							) {
-								positionTop = height + item.height / 2;
+								const theLine = item.element.querySelector(
+									".kit-filtration-group__label-line"
+								) as HTMLDivElement;
+
+								positionTop = theLine.offsetTop +
+									theLine.offsetHeight / 2 - widthBracket;
```

Comment:
Я вот сначала прочитал `withBracket` и подумал "зачем тут отнимается булево значение".
Лучше более грамотно назвать существительное
```suggestion
									theLine.offsetHeight / 2 - bracketWidth;
```